Assets/MixedRealityToolkit/BuildAndDeploy/Editor/BuildDeployWindow.cs

```diff
@@ -1,19 +1,22 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Copyright (c) Rafael Rivera.
```

Comment:
Here's the original file headers for this file: https://github.com/Microsoft/MixedRealityToolkit-Unity/commit/98daf02edf891291adbe33746900513df7ee5c6a#diff-82bf36d076c9b3905af808f0405ca9fdR1

Here's the changes that @riverar made when he added his header https://github.com/Microsoft/MixedRealityToolkit-Unity/commit/e4dcc9b7cb36b5d18a344bda5547b2fb7748bdee.

I can say definitively that those contributions are no longer being used. The XDAGuestLocator was not being used  and it has been removed.

Just for clarification about the rest of the commits to this file made by Rafael, just in case there's any issue:
- https://github.com/Microsoft/MixedRealityToolkit-Unity/commit/271b5bda8af6ebcc40b2be00a67cdd5e13fef662
- https://github.com/Microsoft/MixedRealityToolkit-Unity/commit/e4dcc9b7cb36b5d18a344bda5547b2fb7748bdee
- https://github.com/Microsoft/MixedRealityToolkit-Unity/commit/21ea193afef910920e32d1dc7e762280dc8b787a

I can say definitively that none of the changes are in this file today.  The BuildDeployWindow has made may changes to both it's GUI and underlying logic.

That being said, Rafael if you'd like to re-implement the XDAGuestLocator please feel free to do so and attribute yourself according to any licensing rules set forth my Microsoft.

CC: @davidkline-ms, @keveleigh

Reply:
It wasn't working last time I checked. If you'd like to do a quick test on the master branch, it'll give a definitive answer.

Reply:
OK, will reimplement after BUILD week. Thanks for the heads up!

Reply:
Awesome, I'm looking forward to seeing it, and you at build. 😊